src/gluonts/evaluation/_base.py

```diff
@@ -31,9 +31,22 @@
 import numpy as np
 import pandas as pd
 
+from gluonts.evaluation.metrics import (
```

Comment:
I think relative imports would make more sense here. So `from .metrics import ...`

Reply:
Sure thing, changed it!